az-digital/az_quickstart
# Closes #2786: Update phonebook/directory links in global footer.

## Description
Update global footer with links to directory and phonebook.
Includes DB update.

## Related issues
Closes #2786 

## How to test
1. Enable global footer
2. Ensure "Resources" links include "Directory" and "Phonebook" with correct URLs

Testing on site with global footer already enabled: https://iss-2786-4-dana-qs2.pantheonsite.io/

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
